src/live-stream.js

```diff
@@ -650,35 +651,42 @@ class LiveStream {
     });
   }
   _playIfReadyOGVFinal () {
-    virtualclass.liveStream.ogvPlayer.play();
+    this.ogvPlayer[this.randmoString].play();
   }
   
 
   destroyOGVPlayer() {
     console.log('DESTROY OGV PLAYER');
-    if (this.ogvPlayer) {
-      this.ogvPlayer._stopPlayback();
-      this.ogvPlayer.stop();
+    if (this.ogvPlayer && this.ogvPlayer[this.randmoString]) {
+      this.ogvPlayer[this.randmoString]._stopPlayback();
+      this.ogvPlayer[this.randmoString].stop();
     }
-    delete this.ogvPlayer;
+    delete this.ogvPlayer[this.randmoString];
     delete virtualclass.liveStream.listStream;
     virtualclass.liveStream.listStream = {};
 
-    // this.ogvPlayer.stop();
+    // this.ogvPlayer[this.randmoString].stop();
     delete this.ogvPlayerReady;
+    delete this.randmoString;
     const ogvVideoContainer = document.getElementById('ogvVideoContainer');
     if (ogvVideoContainer != null) {
       ogvVideoContainer.parentNode.removeChild(ogvVideoContainer);
     }
   }
 
   readyOGVInstance() {
-   // if  (!this.ogvPlayer ) {
+   // if  (!this.ogvPlayer[this.randmoString] ) {
       const playerOptions = { forceWebGL: true, debug: false};
-      this.ogvPlayer = new OGVPlayer(playerOptions);
+      if (!this.ogvPlayer) {
+        this.ogvPlayer = {};
+      }
```

Comment:
Codacy found an issue: [Expected indentation of 8 spaces but found 6.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6068204)